Model/ReachConfigProvider.php

```diff
@@ -61,6 +61,7 @@ public function getConfig()
         $config['reach']['dhl_quote']=$this->getDhlQuote();
         $config['reach']['fingerprint_url']=$this->getFingerPrintUrl();  
         $config['reach']['dhl_label']=$this->reachHelper->getDhlDutyLabel();
+        $config['reach']['api_url']=$this->reachHelper->getStashApiUrl();
```

Comment:
I would prefer for this to be ['reach']['stash_url'] or 'stash_api_url' rather than just 'api_url' since that could get confusing, just a naming thing

Reply:
We had a team discussion on Slack about this and I second Lizzy.

Reply:
Variable name has been changed.